frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -831,6 +827,20 @@
             vm.loaderTitle = '';
             vm.newContainer = angular.element('.new-team-card');
 
+            // show loader
+            vm.startLoader = function(msg) {
```

Comment:
Can you please tell us why are you creating this function?

Reply:
because the functions `startLoader` and `stopLoader` which are defined in `frontend/src/js/services/loaderService.js` are toggling `isExistLoader` which is for the left loader not for the right loader, so for right loader in `frontend/src/views/web/challenge/participate.html` I've given the `ng-show="challenge.isLoader"`, therefore this function which I'm creating here just toggling the `isLoader` for the right loader.

Reply:
@RishabhJain2018

Reply:
okay! but it can easily be done by removing right loader code in ` frontend/web/views/participate.html`. There is no need to add this function.
Please let me know if I am not clear.

Reply:
@RishabhJain2018 As I know that we need two loader one when we create new team(displayed on the right side) and another when we go for another page for listing the team(displayed on the left side), so according to me if we want to display two loaders at different places then we need that code or if we want only single loader in both cases to display at center(or any particular place) then we don't need that code in `frontend/src/views/web/challenge/participate.html`

Reply:
We would ideally want a single loader in the center for both the cases you are talking. That's why I was asking to remove the code and have a single loader only.

Reply:
@RishabhJain2018 I've done as you have told me, Please review.